src/main/resources/sqlScripts/client.sql

```diff
@@ -0,0 +1,13 @@
+create table client
+(
+  id          bigint not null
+    constraint client_pkey
+    primary key,
+  name        varchar(255),
+  phone       varchar(255),
+  contact     varchar(255),
+  email       varchar(255) unique,
+  manager_id  bigint constraint client_manager_fkey references user_details);
```

Comment:
Let's also remove manager here and create script, which creates some 'old client' for all current orders.